### PR TITLE
Allow manual deployment of services

### PR DESCRIPTION
**Pull request by berendt**
Signed-off-by: Christian Berendt <berendt@osism.tech>

### CODE REVIEW

File: doc/source/deployment.rst [hunk 1]
```diff
@@ -34,3 +34,9 @@ The ``Run part 3`` task takes some time to complete, depending on the cloud. Run
 
    Path to the ``ansible-galaxy`` binary or the ``ansible-playbook`` only needs to be
    adjusted if the binaries are not findable via ``PATH``.
+
+.. note::
+
+   Add ``-e manual_deploy=true`` if only the necessary infrastructure should be create.
```
Comment (osfrickler) [inline doc/source/deployment.rst:40]: created

File: doc/source/deployment.rst [hunk 2]
```diff
@@ -34,3 +34,9 @@ The ``Run part 3`` task takes some time to complete, depending on the cloud. Run
...
+   Other services such OpenStack or Ceph are then not deployed and can be added manually
```
Comment (osfrickler) [inline doc/source/deployment.rst:41]: such as

**Comment by berendt** [comment]
recheck testbed-deploy

**Review by osfrickler — APPROVED** [review]
Failure was unrelated, merging anyway

**Pull request merged**